### PR TITLE
Add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by Mugiira-Kim**
**In this pair programming project we were able to:**

Add contact form in the footer
Include name field that accepts up to 30 chars.
Include an email field accepts only emails.
Include message field accepts up to 500 chars.
Give input fields placeholder text.
Make sure users cannot submit the form if one or more of the input fields is empty.
Collect data by using the Formspree service.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -491,4 +492,84 @@ aside {
...
   }
+
+  .footer {
+    position: absolute;
+    background-image: url(Contact\ me.png);
+    top: 599%;
+    padding: 50px 6px 0;
+    width: fit-content;
+    overflow: hidden;
+  }
```
Comment (vic778) [inline style.css]: kindly fix the padding of your form. please You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

**Review by vic778 — CHANGES_REQUESTED** [review]
## Hello @Mugiira-Kim 

Good job so far! I know you have worked hard on making this project.  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [x] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Mugiira-Kim and @MoRaad97, 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good PR and Description



 # Required Changes ♻️

- [ ] As the previous reviewer requested kindly make sure your design matches that of the design given, currently its missing an image as shown below:

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image | image



_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @Mugiira-Kim

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

- [ ] [OPTIONAL] It will be nice if you make the page more responsive by removing the space at the right 
- [ ] [OPTIONAL] You did a good job creating the design, but there are still some changes that will make your design even better, you can check the padding between the  message input and the button

Screenshot_20220427_143903


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**